src/main/java/io/electrum/prepaidutility/api/TokenPurchasesResource.java

```diff
@@ -83,6 +83,17 @@
       }
    }
 
+   public class TrialTokenPurchaseRequest {
+      public static final String TRIAL_TOKEN_PURCHASE_REQUEST = "trialTokenPurchaseRequest";
+      public static final int SUCCESS = 200;
+      public static final String RELATIVE_PATH = "/{" + PathParameters.PURCHASE_ID + "}/trials";
+      public static final String PATH = TokenPurchasesResource.PATH + RELATIVE_PATH;
+
+      public class PathParameters {
+         public static final String PURCHASE_ID = "purchaseId";

```

Comment:
as above, we don't really want the ID of the _purchase_ in the trial:
```suggestion
         public static final String REQUEST_ID = "requestId";
```